jenkins/jobs/kata-containers-2.0-centos-PR/config.xml

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
@GabyCT - Travis failed due to this blank line, so I think it's safest if you remove it:

```
INFO: Checking XML file 'jenkins/jobs/kata-containers-2.0-centos-PR/config.xml'
-:2: parser error : XML declaration allowed only at the start of the document
<?xml version='1.0' encoding='UTF-8'?>
     ^
ERROR: failed to check XML file 'jenkins/jobs/kata-containers-2.0-centos-PR/config.xml'
```

Reply:
done :)